tracer/test/test-applications/integrations/Samples.AzureServiceBus/README.md

```diff
@@ -0,0 +1,186 @@
+## Instrumentation behaviors
+Before reading on, make sure you understand the current instrumentation behavior to make sense of the generated traces from `Azure.Messaging.ServiceBus`:
+
+### ServiceBusSender.SendMessageAsync and ServiceBusSender.SendMessagesAsync
+When this API is called, the following diagnostic activity occurs:
+1. A `Message` Activity is started, whose TraceId/SpanId is injected into the Azure Service Bus message's properties `Diagnostic-Id` and `traceparent` (and `tracestate` property if applicable)
```

Comment:
Is this correct:

```suggestion
1. A `Message` Activity is started, whose TraceId/SpanId is injected into the Azure Service Bus message's properties `Diagnostic-Id` and `traceparent` (and `tracestate` property if applicable). The message activity is then stopped.
```

Reply:
Thanks! Done in 46542c7eccdecdbb612460fde446cbc4ce3c2cdf